panoramapublic/src/org/labkey/panoramapublic/query/ExperimentAnnotationsTableInfo.java

```diff
@@ -430,6 +385,19 @@ private ExprColumn getCatalogEntryCol()
         return col;
     }
 
+    private ExprColumn getIsPublicCol()
+    {
+        SQLFragment isPublicColSql = new SQLFragment(" (SELECT CASE WHEN EXISTS (SELECT 1 FROM ")
```

Comment:
This probably won't work if a container is inheriting permissions from its parent. That may not be relevant for this use case though.

Reply:
I should have noted that these containers do not inherit permissions from parent, so that is not a concern.